src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs

```diff
@@ -202,6 +214,10 @@ static void SetStroke(in ICanvas canvas, in float lineWidth, in Color lineColor)
 		{
 			canvas.StrokeColor = lineColor;
 			canvas.StrokeSize = lineWidth;
+			canvas.StrokeDashOffset = 0;
```

Comment:
explicitly set to default values to make sure user doesn't override it in DrawAction